Makefile

```diff
@@ -149,11 +149,11 @@ build: build/llvm.BUILT build/wasi-libc.BUILT build/compiler-rt.BUILT build/libc
 strip: build/llvm.BUILT
 	cd $(PREFIX)/bin; strip clang-9 lld llvm-ar
 
-package: build/package.BUILT
+package: dist
 
-build/package.BUILT: build strip
-	command -v dpkg-deb >/dev/null && ./deb_from_installation.sh || true
-	./tar_from_installation.sh
-	touch build/package.BUILT
+dist: build strip
+	mkdir -p dist
+	command -v dpkg-deb >/dev/null && ./deb_from_installation.sh $(shell pwd)/dist || true
+	./tar_from_installation.sh $(shell pwd)/dist
```

Comment:
Why not continue to use stamp files for consistency?  i.e. can you move into the "dist" directory with out changing the way `build/package.BUILT` is used.   Having `package` depend a `dist` seems add because `dist` in this case is a directory rather than a `.phony` targets.

Reply:
Fixed, thanks